es6.md

```diff
@@ -145,3 +145,49 @@ interface ClassExpression <: Class, Expression {
     type: "ClassExpression";
 }
 ```
+
+# Modules
+
+## ImportDeclaration
+
+**FIXME:** Esprima's behavior differs from Acorn's and SpiderMonkey's behavior in that `source` is of type `ModuleSpecifier`, but is otherwise a `Literal`.
```

Comment:
Is this necessary? The point of ESTree is to remove these discrepancies instead of documenting them.